### ISSUE TITLE
Inserir botão "Concluir" na conexão

### ISSUE THREAD

**Issue by Victormwlopes** [rafaelcorreiapoli/shact#36]
Os usuários ficam sem saber o que fazer quando acaba uma conexão. Um botão que retorna à tela inicial

**Comment by rafaelcorreiapoli** [comment]
Pode tirar umas print's me explicando melhor a questão ?

**Comment by rafaelcorreiapoli** [comment]
@Victormwlopes , pode explicar melhor?

**Comment by Victormwlopes** [comment]
Agora, logo após eu marcar uma pessoa, para eu voltar e tirar outra foto, ou fazer qualquer outra coisa dentro do app, eu preciso apertar voltar. Apertando voltar, abre de novo a câmera do celular, e só depois após sair da câmera, volta para o cenário inicial. A idéia seria inserir um botãozinho "Concluir" na tela ao final das conexões. Ou se não for possível/for muito difícil, pode estar lá o tempo inteiro. Isso não faz muita diferença. Apenas para gerar uma Conclusão no processo de conexão
![image](https://cloud.githubusercontent.com/assets/15360568/12417971/248c91ca-be93-11e5-87a7-27295630ac80.png)

**Comment by Victormwlopes** [comment]
Algo que ficaria embaixo do nome da pessoa após ela ser identificada na foto

**Comment by rafaelcorreiapoli** [comment]
Entao, o unico problema que vejo eh que esta tela pode ser acessada dps pelo menu de fotos, e ai nao faria sentido ese botao. Soh se fizer assim, soh aparece esse botao caso o cara chegue nessa tela apos ter usado o mecanismo de selfie. Tendo esse botao de concluir, faz sentido o botao voltar retornar à camera, né? Que que vc acha?

**Comment by rafaelcorreiapoli** [comment]
to vendo por aqui, sua versao ta desatualizada, ta com a appbar grande e a status bar preta. Esses negocios nao atualizam sozinho, precisa recompilar ai no seu cel. Quando vc tiver com a versao atualizada c vai ver que bunito

**Comment by Victormwlopes** [comment]
Sim! Se der para fazer isso, é perfeito. Parece coisa besta, mas pensando na jornada de quem usa o app, precisa ter um início meio e fim, sabe? Tipo uma fase de um video game. 
O botão "Voltar" está perfeito ali, porque as vezes a pessoa não foi identificada na foto, precisa tirar outra, sei lá, aí faz sentido voltar pra câmera. Mas para outras situações, precisa do botão concluir.

**Comment by Victormwlopes** [comment]
E boa! Vou ver se consigo atualizar sozinho aqui com meu pc
